OrbitGl/Batcher.cpp

```diff
@@ -163,10 +159,14 @@ void Batcher::GetBoxGradientColors(const Color& color,
   (*colors)[3] = color;
 }
 
-void Batcher::Reset() {
+void Batcher::Reset(bool start_new_frame) {
```

Comment:
Hi Florian, could this be split into 2 functions, Reset() and ResetAndStartNewFrame(), or Reset() and StartNewFrame()? It would be clearer I think. Passing a boolean as argument is not ideal, another option would be to pass an enum.

Reply:
I've split it into ResetElements() and StartNewFrame() to make this clearer, thanks for pointing this out.